api.swagger.yaml

```diff
@@ -4162,6 +4162,185 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/promotions:write ]
+  '/web_forms':
```

Comment:
Can this be changed to 'webform' to match the product? In endpoint names, category, descriptions etc

Reply:
Changed it all to all one word.
I take it you still want plurals.
EP path: /webforms
category webforms
Thats how it is now.